Xamarin.Forms.Material.Android/FormsInitExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Android.Content;
+using Android.OS;
+
+namespace Xamarin.Forms
+{
+	public static class FormsInitExtensions
+	{
+		public static IFormsInit WithMaterial(this IFormsInit init, Context context, Bundle bundle)
```

Comment:
In this case, it means that it will be VisualMaterial.
but when new styles appear, they too will be able to make extensions with their names.
Like: .WithFabriculous()

Reply:
probably then I need to add some parameter that would allow to specify which Material we need.

Reply:
@jsuarezruiz WithVisualMaterial is ok?

Reply:
would an alternative be to use an Enum perhaps?

```csharp
.WithVisual(Visual.Material)
```

Reply:
@ChaseFlorell there are 2 questions here:
If this enum is a part of Xamarin.Forms, then third-party developers will not be able to change it.
What should I do if I need to add several styles?

Reply:
That's a good point, and likely the reason why Xamarin.Forms went with strings for the On Platform bits. `Device.iOS` and `Device.Android` for example

```csharp
public static class VisualStyle
{
    public const string Material = "Material";
}
```

```csharp
.WithVisual(VisualStyle.Material)
// or
.WithVisual("Material")
```

Reply:
about VisualMeterial
https://github.com/xamarin/Xamarin.Forms/pull/8220/files#diff-fe9c32d1af198b6530d35e46b8d7a8da
it looks like there is initialization of the renderers, and it makes no sense to add enum or lines.
If there will be new visual styles, then their developers will add an extension.
In addition, I think that it will be possible to use several visual styles at once.